apps/system/utils/netcmd.c

```diff
@@ -95,6 +95,95 @@ struct ifconfig_cmd_info_s {
 extern int netdb_main(int argc, char *argv[]);
 #endif
 
+int getIp(void)
+{
+	struct ifreq *ifr;
+	struct sockaddr_in *sin;
+	struct sockaddr *sa;
+	struct ifconf ifcfg;
+	int ret = -1;
+	int fd;
+	int numreqs = 3;
+	int num_nic = 0;
+	int ret1 = 0;
+	struct ifaddrs *ifa = NULL, *ifp = NULL;
+	ret = netlib_getifaddrs(&ifa);
+	if (ret < 0) {
+		return 0;
```

Comment:
I think return ret is better option, some of apis in tizenRT treat '0' as a 'OK' and for error case, platform using negative value as a ERROR